install/terraform/modules/aks/aks.tf

```diff
@@ -63,12 +63,12 @@ resource "azurerm_kubernetes_cluster" "agones" {
   kubernetes_version = "1.18.14"
```

Comment:
`kubernetes_version = "1.20.5"`

1.18.4 deprecated soon.

Reply:
i see, 1.18.* will not help. but as per docs Agones was not tested with higher. I think it should be other PR.

Reply:
Only 1.18 is supported on Agones right now, so we can't go and change the version, without the process to upgrade:
https://agones.dev/site/docs/installation/

Reply:
afaik MS will prevent me to deploy 1.18 cluster this month. i do not yet have production cluster for k8s. so may be I must hedge that by preallocating:) 
what are plans for 1.19 for agones?

Reply:
> Agones will update its support to n-1 version of what is available across the majority of major cloud providers - GKE, EKS and AKS, while also ensuring that all Cloud providers can support that version.
https://agones.dev/site/docs/installation/

From review, we are ready to move to 1.19.

Reply:
I checked at the start of this release cycle and we weren't quite ready. Given how soon the next release is, I was planning to do the update to 1.19 for the following release.